charts/htsget-rs/values.yaml

```diff
@@ -0,0 +1,183 @@
+  #      - htsget.local
+  # issuer requires a configured cert-manager
+  issuer: ""
+  clusterIssuer: ""
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 20
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# htsget-rs specific configuration.
+
+# The application's ticket server TLS can be disabled by setting enabled to false.
+# Not to be used in production but handy for testing.
+tls:
+  enabled: false
+  issuer: ""
+  clusterIssuer: ""
+  secretName: ""
+
+c4gh:
+  # Set to true if htsget will use a predefined c4gh key-pair instead
+  # of creating one on-the-fly when handling encrypted data
+  predefined: false
+  # The name for the secret containing c4gh keys.
+  secretName: ""
+
+# Enable data server TLS
+tlsDataServer:
+  enabled: false
+  secretName: ""
+
+# Enable client TLS for the urlstorage backend (experimental feature)
+tlsClient:
+  enabled: false
+  secretName: ""
+
+htsget:
+  # Logging levels and formatting (inherited by rust)
+  rustLog: "info" # debug, trace
+  formattingStyle: "Pretty" # Full, Compact, Pretty, Json
+
+  # Path where the sensitive files can be found.
+  # These should match the path in the htsget config below.
+  c4ghPath: ""
+  tlsPath: "/tls"
+  tlsPathDataServer: ""
+  tlsPathClient: ""
+
+# The htsget-rs server is configured by placing the contents of the toml config file below (keep nindent 2).
+# All of the available configuration options (https://github.com/umccr/htsget-rs/tree/crypt4gh/htsget-config)
+# can be set this way. Paths to sensitive files like TLS certicifaces or c4gh keys need to be set above and match
+# those used below. The names of the files that are referenced below should be the same as the ones in the respective
+# secrets whose names can be set above.
+# Below is an example of a htsget-rs config file utilizing a url_storage.
+configMapData: |-
+  # Information displayed by the /info endpoint
+  name = "htsget.example"
+  version = ""
+  organization_name = "Example Organization"
+  organization_url = "http://example.com"
+  contact_url = "http://example.com"
+  documentation_url = "https://github.com/umccr/htsget-rs"
+
+  # The ticket server configuration
+  ticket_server_addr = "127.0.0.1:8080"
+  ticket_server_cors_allow_origins = "All"
+  # ticket_server_tls.key = "/tls/tls.key"
+  # ticket_server_tls.cert = "/tls/tls.crt"
+
+  # disable the internal data server
+  data_server_enabled = false
```

Comment:
Done. In this first iteration I had not considered to cover all configuration setups but yes, it makes sense to have the most generic usecase as the main example here instead of the GDI one. I now added logic for enabling local storage for the data server and should allow separating the data server to a different service and ingress I guess.